table_engine/src/table.rs

```diff
@@ -487,6 +487,11 @@ pub trait Table: std::fmt::Debug {
     /// Get table's statistics.
     fn stats(&self) -> TableStats;
 
+    /// Whether the columns used in filter expr can be pushdown....
+    /// `read_schema` is used here to avoid upper layer see different schema
+    /// during one query.
```

Comment:
```suggestion
    /// Whether the columns used in filter expr can be pushdown.
    ///
    /// `read_schema` is used here to avoid upper layer see different schema
    /// during one query.
```